api/nakadi-event-bus-api.yaml

```diff
@@ -606,22 +488,394 @@ paths:
+
+            - `auto_commit`: everything that Nakadi sends to stream is considered to be committed. No actions are
+            required from client side to commit. Be aware that with this commit mode loosing of some events is very
+            probable. So you should know what you are doing if you use autocommit.
+          type: string
+          enum:
+            - commit_at_will
+            - commit_each
+            - auto_commit
+          required: false
+          default: commit_at_will
+        - $ref: '#/parameters/BatchLimit'
+        - $ref: '#/parameters/StreamLimit'
+        - $ref: '#/parameters/BatchFlushTimeout'
+        - $ref: '#/parameters/StreamTimeout'
+        - $ref: '#/parameters/StreamKeepAliveLimit'
+        - $ref: '#/parameters/XFlowId'
+      responses:
+        '200':
+          description: |
+            Ok. Stream started.
+            Stream format is a continuous series of `EventStreamBatch`s separated by `\n`
+          schema:
+            $ref: '#/definitions/EventStreamBatch'
+          headers:
+            X-Nakadi-StreamId:
+              description: |
+                the id of this stream generated by Nakadi. Must be used for committing events that were read by client
+                from this stream.
+              type: string
+        '400':
+          description: Bad Request
+          schema:
+            $ref: '#/definitions/Problem'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Client is not authorized to perform this operation
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: Subscription not found.
+          schema:
+            $ref: '#/definitions/Problem'
+        '409':
+          description: |
+            Conflict. There are no empty slots for this subscriptions. The amount of consumers for this subscription
+            already equals the maximal value - the total amount of this subscription partitions.
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+        '503':
+          description: Service (temporarily) unavailable
+          schema:
+            $ref: '#/definitions/Problem'
+
+  '/registry/validation-strategies':
+    get:
+      tags:
+        - schema-registry-api
+      description: |
+        Lists all of the validation strategies supported by this installation of Nakadi.
+
+        If the EventType creation is to have special validations (besides the default), one can consult over
+        this method the available possibilities.
+      responses:
+        '200':
+          description: Returns a list of all validation strategies known to Nakadi
+          schema:
+            type: array
```

Comment:
> Also, is there something in API guidelines about this point? I think it would be valuable to add there something about that.

There's an open ticket for this https://github.com/zalando/restful-api-guidelines/issues/117 which is being worked on by the guild atm. 

> But if we know for sure that it will be only list - I think it's pretty safe to just return array.

This is true, but the issue then is you end up with an API that has two ways of working based on the size of the result list. That might be easier but it's not simpler. It's much more ideal to have one approach.